pkg/remotewrite/validate.go

```diff
@@ -37,5 +41,6 @@ func ValidateRequest(remoteWriteRequest *prometheus.WriteRequest) (string, error
 		return key, nil
 	}
 
-	return "", errors.New("request does not contain any cluster ids")
+	message := fmt.Sprintf("request does not contain any cluster ids: %v", remoteWriteRequest)
```

Comment:
Instad of creating a new error explicitly we can do:
return "", fmt.Errorf("request does not contain any cluster ids: %v", remoteWriteRequest)